pkg/services/sqlstore/migrations/playlist_mig.go

```diff
@@ -33,11 +45,9 @@ func addPlaylistMigrations(mg *Migrator) {
 		{Name: "value", Type: DB_Text, Nullable: false},
 		{Name: "title", Type: DB_Text, Nullable: false},
 	}))
-}
 
-func addPlaylistUIDMigration(mg *Migrator) {
```

Comment:
^^ this was split as a different function before, but now all migrations added from the same entry point

Reply:
But this also changes the order of the migrations, doesn't it? What's the benefit of merging it into one function?

Reply:
removed the cleanup -- the only reason i did it was to reuse the same table structure created in `playlistV2()` but does not really matter, we can create a new one for every migration